starcheck/src/lib/Ska/Starcheck/Obsid.pm

```diff
@@ -65,6 +65,39 @@ def _get_agasc_stars(ra, dec, roll, radius, date, agasc_file):
 
     return stars_dict
 
+import numpy as np
+import mica.stats.acq_stats
+import mica.stats.guide_stats
+
+ACQS = mica.stats.acq_stats.get_stats()
+GUIDES = mica.stats.guide_stats.get_stats()
+
+
+def get_mica_stats(agasc_id, time):
+
+    acqs = ACQS[ACQS['agasc_id'] == int(agasc_id)]
+    time = float(time)
+    acq_5y = acqs[(acqs['guide_tstart'] >= (time - (5 * 365 * 86400)))
```

Comment:
Why the 5 year time limit?  I think the two important pieces of information are:
- Best estimate of observed mag
- Whether actual p_acq is persistently very different from model p_acq, e.g. due to a nearby bright spoiler.

Did you have a different idea?

Reply:
I didn't "rethink" about this when re-implimenting this last night, and left this mismatch (5y for acq on noid from the original prototype, all data for guide) in this WIP.

For acq, what do you consider a useful way to measure whether p_acq is persistently very different from model p_acq?

Reply:
The super-simple thing to do is just add all the `p_acq` and compare to the actual number that were acquired.  If the probability model is applicable to the star then those values should match.  For extreme cases it is easy, e.g. sum of p_acq is 19 (for e.g. 23 trials) and in fact only 3 acqs were successful.   

The trick is in establishing confidence intervals for not-so-obvious cases.  This is basically like summing bernoulli distributions, but I'm not sure offhand if there is a clean computational formalism.  One could always do Monte-Carlo simulation with a modest number of trials (a few hundred) to find cases of "very difference from the model".